packages/react-native/Libraries/Network/XMLHttpRequest.js

```diff
@@ -656,7 +656,7 @@ class XMLHttpRequest extends (EventTarget(...XHR_EVENTS): any) {
   }
 
   /* global EventListener */
-  addEventListener(type: string, listener: EventListener): void {
+  addEventListener = (type: string, listener: EventListener): void => {
```

Comment:
Not sure if this is the right way to go about this -- ideally I can type `EventTargetStub` such that no source changes are required.

Without arrow function, I get this error
```
annot extend typeof EventTargetStub [1] with XMLHttpRequest because function [2] cannot be unbound from the context [3] where it was defined in the this parameter of property addEventListener. [method-unbinding]
```

Reply:
spotted some `// $FlowFixMe[method-unbinding` elsewhere in the code. Maybe that's what I need here ..